docs/SpendPermissionAccounting.md

```diff
@@ -0,0 +1,68 @@
+# Spend Permission Accounting
+
+## Onchain Accounting
+
+Tracking asset expenditure accurately is important for enforcing user-approved limits. Given the fragility and complications with doing this accounting offchain and the severity of inaccuracies, we designed for fully onchain accounting. Onchain accounting enables us to have higher confidence in its reliability and keep our system trust-minimized.
```

Comment:
the acceptability of offchain accounting seems like such a self-evidently inappropriate design choice that I would actually shy away from even including this rationale here, especially because the context/relationship between this contract and the larger coinbase smart wallet product isn't necessarily obvious here either.  as a more naive reader, this sentence would give me pause and actually lead me to trust everything less. I think we could just keep the first sentence above and delete the second two?